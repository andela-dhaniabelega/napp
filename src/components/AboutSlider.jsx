import React from 'react';
import '../../sass/styles.scss';

const AboutSlider = () => (
  <div className="slider_area">
    <div className="slider">
      <ul className="bxslider">
        <li>
          <img src="../images/about.jpg" alt="" />
        </li>
      </ul>
    </div>
  </div>   
);

export default AboutSlider;

