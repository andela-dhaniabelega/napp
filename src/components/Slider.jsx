import React from 'react';
import '../../public/sass/styles.scss';

/**
 * Stateless component
 * Simply renders a slider
 *
 * @return {void}
 */
const Slider = () => (
  <div className="slider_area">
    <div className="slider">
      <ul className="bxslider">
        <li>
          <img src="../public/images/welcome.png" alt="" />
        </li>
      </ul>
    </div>
  </div>   
);

export default Slider;

