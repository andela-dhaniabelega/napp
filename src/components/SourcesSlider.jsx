import React from 'react';
import '../../public/sass/styles.scss';
import Bar from '../components/Bar';

/**
 * Stateless component
 * Simply renders Slider for Sources component
 *
 * @return {void}
 */
const SourceSlider = () => (
  <div className="slider-margin">
    <Bar />		
    <div className="slider_area">
      <div className="slider">
        <ul className="bxslider">
          <li>
            <img src="../public/images/source.jpg" alt="" />
          </li>
        </ul>
      </div>
    </div>
  </div>   
);

export default SourceSlider;

