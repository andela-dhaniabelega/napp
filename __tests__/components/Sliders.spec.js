import React from 'react';
import renderer from 'react-test-renderer'
import AboutSlider from '../../src/components/AboutSlider';
import SourcesSlider from '../../src/components/SourcesSlider';
import ContactUsSlider from '../../src/components/ContactUsSlider';
import Slider from '../../src/components/Slider';

describe('Sliders', () => {
	describe('About Slider', () => {
		it('renders a snapshot', () => {
          const tree = renderer.create(<AboutSlider />).toJSON();
          expect(tree).toMatchSnapshot();
		});
    });

    describe('Source Slider', () => {
		it('renders a snapshot', () => {
          const tree = renderer.create(<SourcesSlider />).toJSON();
          expect(tree).toMatchSnapshot();
		});
    });

    describe('Contact Us Slider', () => {
		it('renders a snapshot', () => {
          const tree = renderer.create(<ContactUsSlider />).toJSON();
          expect(tree).toMatchSnapshot();
		});
    });

    describe('Home Slider', () => {
    it('renders a snapshot', () => {
          const tree = renderer.create(<Slider />).toJSON();
          expect(tree).toMatchSnapshot();
    });
    });
});