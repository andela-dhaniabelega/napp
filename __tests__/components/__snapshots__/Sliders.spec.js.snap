// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sliders About Slider renders a snapshot 1`] = `
<div
  className="slider-margin"
>
  <div
    className="main_menu_area"
  >
    <ul
      id="nav"
    >
      <span
        className="u-id"
      />
    </ul>
  </div>
  <div
    className="slider_area"
  >
    <div
      className="slider"
    >
      <ul
        className="bxslider"
      >
        <li>
          <img
            alt=""
            src="../public/images/about.jpg"
          />
        </li>
      </ul>
    </div>
  </div>
</div>
`;

exports[`Sliders Contact Us Slider renders a snapshot 1`] = `
<div
  className="slider-margin"
>
  <div
    className="main_menu_area"
  >
    <ul
      id="nav"
    >
      <span
        className="u-id"
      />
    </ul>
  </div>
  <div
    className="slider_area"
  >
    <div
      className="slider"
    >
      <ul
        className="bxslider"
      >
        <li>
          <img
            alt=""
            src="../public/images/contact.jpg"
          />
        </li>
      </ul>
    </div>
  </div>
</div>
`;

exports[`Sliders Home Slider renders a snapshot 1`] = `
<div
  className="slider_area"
>
  <div
    className="slider"
  >
    <ul
      className="bxslider"
    >
      <li>
        <img
          alt=""
          src="../public/images/welcome.png"
        />
      </li>
    </ul>
  </div>
</div>
`;

exports[`Sliders Source Slider renders a snapshot 1`] = `
<div
  className="slider-margin"
>
  <div
    className="main_menu_area"
  >
    <ul
      id="nav"
    >
      <span
        className="u-id"
      />
    </ul>
  </div>
  <div
    className="slider_area"
  >
    <div
      className="slider"
    >
      <ul
        className="bxslider"
      >
        <li>
          <img
            alt=""
            src="../public/images/source.jpg"
          />
        </li>
      </ul>
    </div>
  </div>
</div>
`;
